packages/components-storybook/package.json

```diff
@@ -15,7 +15,8 @@
         "type-check": "tsc --project tsconfig.types.json",
         "type-check:watch": "yarn type-check -- --watch",
         "ci:test:snapshots": "cypress run",
-        "ci:test:snapshots:references": "cypress run --env updateSnapshots=true customSnapshotsDir=./snapshots"
+        "ci:test:snapshots:references": "cypress run --env updateSnapshots=true customSnapshotsDir=./snapshots",
+        "ci:test:open": "cypress open"
```

Comment:
this does not help much without specifying url to the cypress script. It might be done this way (most likely :D) --config baseUrl=http://localhost:3000

Reply:
> this does not help much without specifying url to the cypress script. It might be done this way (most likely :D) --config baseUrl=http://localhost:3000

```CYPRESS_baseUrl=https://suite.corp.sldev.cz/components-storybook/develop yarn workspace @trezor/components-storybook ci:test:open```